public/js/reader.js

```diff
@@ -316,6 +349,10 @@ Reader.updateMetadata = function () {
 
     const width = img.naturalWidth;
     const height = img.naturalHeight;
+    const width_doublepage = img_doublepage.naturalWidth;
+    const height_doublepage = img_doublepage.naturalHeight;
+    const width_view = width + width_doublepage;
+    const height_view = height > height_doublepage ? height : height_doublepage;
```

Comment:
This constant doesn't seem to be used.

Reply:
this constant use when caculate file width and height size
but i just copy your code single page width height size code